diff -ur Makefile.orig Makefile > Dock-6.6_make-install-fix.patch
--- Makefile.orig	2013-02-04 10:19:15.918563930 +0100
+++ Makefile	2013-02-04 10:21:07.503563314 +0100
@@ -222,3 +222,7 @@
 	cp anchor_and_grow.in rigid.in \
 		../../../tutorials/ligand_sampling_demo/4_dock
 
+
+myinstall:
+	echo mv ../bin ${EASYBUILDINSTALLPATH}/software/Dock/6.6-ictce-4.0.6/ >/tmp/checkmeplease
+	mv ../bin ${EASYBUILDINSTALLPATH}/software/Dock/6.6-ictce-4.0.6/ >/tmp/checkmeplease
